src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -508,4 +516,45 @@ public VisualEditorPageObject clickInfoboxPopup() {
     return this;
   }
 
+  public VisualEditorPageObject addVideoToContent(String videoType) {
+    verifyVEToolBarPresent();
+    verifyEditorSurfacePresent();
+    VisualEditorAddMediaDialog mediaDialog = clickVideoButton();
+    VisualEditorPageObject ve = mediaDialog.addMediaByURL(videoType);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "ve". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1488)